mParticle-Pilgrim.podspec

```diff
@@ -0,0 +1,20 @@
+Pod::Spec.new do |s|
+    s.name             = "mParticle-Pilgrim"
+    s.version          = "7.7.3"
+    s.summary          = "Pilgrim integration for mParticle"
+
+    s.description      = <<-DESC
+                       This is the Pilgrim integration for mParticle.
+                       DESC
+
+    s.homepage         = "https://www.mparticle.com"
+    s.license          = { :type => 'Apache 2.0', :file => 'LICENSE' }
+    s.author           = { "mParticle" => "support@mparticle.com" }
+    s.source           = { :path => "/Users/creyes/Workspace/mparticle-apple-integration-example" }
+    s.social_media_url = "https://twitter.com/mparticles"
+
+    s.ios.deployment_target = "9.0"
+    s.ios.source_files      = 'mParticle-Pilgrim/*.{h,m}'
+    s.ios.dependency 'mParticle-Apple-SDK', '~> 7.8.4'
+    s.ios.dependency 'Pilgrim', '~> 2.1.2'
```

Comment:
Forgot to mention this earlier, you can change to `~> 2.1` to allow minor version updates to be pulled in, otherwise it will not allow e.g. version 2.2.0 to be used.

Reply:
ohh gotcha, sounds good!